devices/MiHumidifierCBL.js

```diff
@@ -0,0 +1,16 @@
+const MiHumidifierCAL = require('./MiHumidifierCAL');
+
+module.exports = class extends MiHumidifierCAL {
+    constructor() {
+        super();
+        this.version = "cbl";
```

Comment:
Correct version is "cb1":

```suggestion
        this.version = "cb1";
```

Also the name of file in this case should be "MiHumidifierCB1"

Reply:
Fixed